src/utils/dates.js

```diff
@@ -10,3 +10,11 @@ const ONE_DAY_IN_MILLISECONDS = 86400000;
 export function getFutureDate(offset) {
 	return new Date(Date.now() + offset * ONE_DAY_IN_MILLISECONDS);
 }
+
+export function getDaysBetweenDates(startTime, currentTime) {
+	// param both in ms
+	const daysSinceLastPurchase = Math.round(
+		(currentTime - startTime) / ONE_DAY_IN_MILLISECONDS,
+	);
+	return daysSinceLastPurchase;
+}
```

Comment:
I attempted to refactor / shorten this, but every attempt reduced readability and was more difficult to follow

I think how you coded this is perfect and the most readable option!

Reply:
awesome! so fresh and so clean clean!